### PR TITLE
Resolve merge conflicts (PR #128)

### PR DESCRIPTION
**Pull request by journey2w**
- Review PR #128 
- Merge s-k-song's Match-URL-path-in-Web-documents branch.
- Resolve conflicts
- Fix some errors (Note tag, links and TOC)

### CODE REVIEW

File: docs/application/native/guides/app-management/efl-ui-app.md [hunk 1]
```diff
@@ -261,4 +262,4 @@ An application with a background running capability must declare the background
 ```
 The `<metadata key="http://tizen.org/metadata/bacgkround-category/<value>"/>` element has no effect on Tizen 2.3 devices, but on Tizen 2.4 and higher devices, it has the same effect as the `<background-category>` element.
 
-The background category of your application can be specified in the [application project settings](../../../../org.tizen.training/html/native/process/setting_properties_n.htm#manifest) in the Tizen Studio.
\ No newline at end of file
+The background category of your application can be specified in the [application project settings](../../../../org.tizen.training/html/native/process/setting_properties_n.htm#manifest) in the Tizen Studio.
```
Comment (mijin85cho) [inline docs/application/native/guides/app-management/efl-ui-app.md]: rf) `../../../tutorials/process/setting-properties.md` would be correct

File: docs/application/tizen-studio/index.md [hunk 2]
```diff
@@ -13,15 +13,15 @@ To develop your applications, use the set of tools provided by the Tizen Studio:
 
 - [Setup](setup/installing-sdk.md) Learn about installing and updating the Tizen Studio. You may need to install additional packages to develop applications for various platforms.
 
-- [Native Tools ](native-tools/cover-native-n.md)Learn about the available native application development tools. Native applications are developed using the C programming language. You can create an application project, edit the code to meet your needs, and run the application on the emulator or a target device.
+- [Native Tools](native-tools/cover-native.md)Learn about the available native application development tools. Native applications are developed using the C programming language. You can create an application project, edit the code to meet your needs, and run the application on the emulator or a target device.
```
Comment (mijin85cho) [inline docs/application/tizen-studio/index.md]: Wouldn't it better to have a space between `[Native Tools](native-tools/cover-native.md)` and `Learn`?

File: docs/application/tizen-studio/index.md [hunk 3]
```diff
@@ -13,15 +13,15 @@ To develop your applications, use the set of tools provided by the Tizen Studio:
...
 
-- [Web Tools ](web-tools/cover-web-w.md)Learn about the available Web application development tools. Web applications are developed using the JavaScript, HTML, and CSS languages. You can create an application project, edit the code to meet your needs, and run the application on the emulator, simulator, or a target device.
+- [Web Tools](web-tools/cover-web.md)Learn about the available Web application development tools. Web applications are developed using the JavaScript, HTML, and CSS languages. You can create an application project, edit the code to meet your needs, and run the application on the emulator, simulator, or a target device.
```
Comment (mijin85cho) [inline docs/application/tizen-studio/index.md]: Wouldn't it better to have a space between `[Web Tools](web-tools/cover-web.md)` and `Learn`?

File: docs/application/tizen-studio/index.md [hunk 4]
```diff
@@ -13,15 +13,15 @@ To develop your applications, use the set of tools provided by the Tizen Studio:
...
 
 - [Platform Tools](platform-tools/cover-platform.md) Learn about the platform development environment that helps you develop preloaded platform applications, platform libraries, and device drivers in Ubuntu.
 
-- [Tizen Studio for RT](rt_sdk/rt_cover.md) Learn about the Tizen Studio for RT and how you can develop lightweight RTOS-based applications.
+- [Tizen Studio for RT](rt-sdk/rt-cover.md) Learn about the Tizen Studio for RT and how you can develop lightweight RTOS-based applications.
```
Comment (mijin85cho) [inline docs/application/tizen-studio/index.md]: `rt-ide/rt-overview.md` looks correct.

File: docs/application/web/getting-started/wearable/first-app.md [hunk 5]
```diff
@@ -235,15 +235,15 @@ To run the application on the emulator:
...
 
         While the application is running, the **Log** view in the Tizen Studio shows the log, debug, and exception messages from the methods defined in the log macros. To see the view, in the Tizen Studio menu, go to **Window &gt; Show View &gt; Log**.
 
 For more information on using the emulator features, see [Using Emulator Control Keys, Menu, and Panel](../../../tizen-studio/common-tools/emulator-control-panel.md) and [Using Extended Emulator Features](../../../tizen-studio/common-tools/emulator-features.md).
 
 <a name="target"></a>
-### Running on a Target Device 
+### Running on a Target Device
 
 To run the application on a target device:
```
Comment (mijin85cho) [inline docs/application/web/getting-started/wearable/first-app.md]: Don't we need <a name="remote_device"></a> in this page on line 270?

File: docs/application/web/getting-started/wearable-watch/first-app-watch.md [hunk 6]
```diff
@@ -260,7 +260,7 @@ To run the application on the emulator:
 
         ![Application running in the emulator](media/emulator_running_watch_ww.png)
 
-        > **Note** 
+        > **Note**  
 		> If the emulator display has switched off, you cannot see the application launch. To switch the display on, click the **Power** key (in the lower-right corner of the emulator).
 
     3.  Change the watch face.
```
Comment (mijin85cho) [inline docs/application/web/getting-started/wearable-watch/first-app-watch.md]: Don't we need <a name="watchface"></a> here?

File: docs/application/native/guides/app-management/efl-ui-app.md (hunk 1 above)
Comment (journey2w) [reply docs/application/native/guides/app-management/efl-ui-app.md]: The file name is still ../../tutorials/process/setting-properties-n.md.
Removing '-n' from filename under native directory is done by other members.

Moreover, I'm checking wrong links and modinging TOC.md on https://github.com/journey2w/tizen-docs/tree/dotnet-toc/docs.
I will double check this link error after removing '-n' from filename.

File: docs/application/tizen-studio/index.md (hunk 2 above)
Comment (journey2w) [reply docs/application/tizen-studio/index.md]: I'm modifying index.md and TOC.md on https://github.com/journey2w/tizen-docs/tree/dotnet-toc/docs.

I  will change file names and directory paths under tizen-studio directory.

File: docs/application/tizen-studio/index.md (hunk 3 above)
Comment (journey2w) [reply docs/application/tizen-studio/index.md]: I'm modifying index.md and TOC.md on https://github.com/journey2w/tizen-docs/tree/dotnet-toc/docs.

File: docs/application/tizen-studio/index.md (hunk 4 above)
Comment (journey2w) [reply docs/application/tizen-studio/index.md]: I'm modifying index.md and TOC.md on https://github.com/journey2w/tizen-docs/tree/dotnet-toc/docs.

File: docs/application/web/guides/TOC.md [hunk 7]
```diff
@@ -1,196 +1,197 @@
 # Application Management
 ## Applications
-### [Service Application](app-management/service-app-w.md)
-### [Watch Application](app-management/watch-app-w.md)
-### [Widget Application](app-management/web-widget-ww.md)
-## [Application Groups](app-management/app-group-w.md)
-## [Application Preferences](app-management/preference-w.md)
-## [Application Icons](app-management/app-icons-w.md)
+### [Service Application](app-management/service-app.md)
+### [Watch Application](app-management/watch-app.md)
```
Comment (mijin85cho) [inline docs/application/web/guides/TOC.md]: absent page : This watch-app.md page is left out by the author, so we don't have this page on git. It only exists on Tizen Developer Site. Thus, this link is broken.

File: docs/application/web/guides/TOC.md [hunk 8]
```diff
@@ -1,196 +1,197 @@
...
 
 # Security
-## [Secure Key Management](security/secure-key-w.md)
+## [Secure Key Management](security/secure-key.md)
 
-# [Error Handling](error/error-w.md)
+# [Error Handling](error/error.md)
 
-# [W3C/HTML5/Supplementary Features](w3c/w3c-cover-w.md)
+# [W3C/HTML5/Supplementary Features](w3c/w3c-cover.md)
```
Comment (mijin85cho) [inline docs/application/web/guides/TOC.md]: w3c-overview.md looks correct.

File: docs/application/web/guides/TOC.md [hunk 9]
```diff
@@ -1,196 +1,197 @@
...
-### [Performance Improvement](w3c/useful/performance-w.md)
+### [Tizen WebKit Tap Sound Policy](w3c/useful/sound-policy.md)
+### [Performance Improvement](w3c/useful/performance.md)
 ## Supplementary Features
-### [Overview](w3c/supplement/supplement-w.md)
-### [Typed Array - Khronos](w3c/supplement/typedarray-w.md)
-### [WebGL - Khronos](w3c/supplement/webgl-w.md)
-### [FullScreen API - Mozilla](w3c/supplement/fullscreen-w.md)
-### [Camera API (Tizen Extension)](w3c/supplement/camera-w.md)
+### [Overview](w3c/supplement/supplement.md)
```
Comment (mijin85cho) [inline docs/application/web/guides/TOC.md]: supplement-guide.md is correct.

File: docs/application/web/guides/app-management/app-controls.md [hunk 10]
```diff
@@ -139,23 +133,27 @@ The Tizen common application controls specify a standard protocol for sharing ap
...
+- [Input Delegator](./common-appcontrol.md#input-delegator)
+- [Map](./common-appcontrol.md#map)
+- [Message](./common-appcontrol.md#message)
+- [Multimedia](./common-appcontrol.md#multimedia)
+- [System Settings](./common-appcontrol.md#system-settings)			  
+  * [Settings for Bluetooth](./common-appcontrol.md#settings-for-bluetooth)
+  *	[Settings for Location](./common-appcontrol.md#settings-for-location)
+  *	[Settings for NFC](./common-appcontrol.md#settings-for-nfc)
+  *	[Settings for Wi-Fi](./common-appcontrol.md#settings-for-wi-fi)
+- [Voice Recorder](./app-management/common-appcontrol.md#voice-recorder)
```
Comment (mijin85cho) [inline docs/application/web/guides/app-management/app-controls.md]: Should remove this `/app-management`.

File: docs/application/web/guides/app-management/app-management-cover.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
+# Application Management
+
+The application management features cover the application models available for Web applications. They describe the application life-cycle and resources, and the communication methods available for the application.
+
+You can use the following application management features in your Web applications:
+
+- [Applications](./applications.md)  
+You can manage applications and get information about them. You can also handle events and enable an application to run in the background.
+
+ Tizen provides various application models to allow you to create applications targeted for specific tasks. You can create UI applications to run on the foreground, and service applications to run in the background. The service application is used to perform tasks that need to run periodically or continuously, but do not require any user interaction.
```
Comment (mijin85cho) [inline docs/application/web/guides/app-management/app-management-cover.md]: minor) This paragraph is sub item of the list, under `Application`. However it is out of the list line now.

File: docs/application/web/guides/app-management/app-management-cover.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+You can set up application badges. In the device application list, you can show a badge with the application icon to provide additional information about the application state or notifications to the user.
+
+- [Application Information and Controls](./app-info-control.md)  
+The application controls allow you to share an application's functionality. Using another application's features through application controls reduces the time and effort needed to develop your own application. You can also manage and retrieve information from installed widgets and widget instances in various ways.
+
+- [Application Data Exchange](./app-communication.md)  
+The application can communicate with other applications in various ways. You can exchange data between applications through message ports and data controls.
+
+- [Package Information](./packages.md)  
+You can retrieve detailed information about application packages. You can receive notifications if packages are updated or removed, or new packages are installed. You can also perform package management tasks, such as installing or uninstalling packages.
```
Comment (mijin85cho) [inline docs/application/web/guides/app-management/app-management-cover.md]: `Dependencies` is missing.

File: docs/application/web/guides/app-management/web-widget.md [hunk 13]
```diff
@@ -1430,90 +1440,116 @@ var parent = document.getElementById('parentDiv');
...
 
+<a name="faq29"></a>
 #### What type of content is best for Web widgets?
 
 Typically as part of a Web application, Web widgets provide their users with important Web application functions (such as launching the parent Web application), and brief, important Web application information. Therefore, simple and lightweight services are best suited for Web widgets.
 
+<a name="faq30"></a>
 #### Can I use CSS `ellipsis`?
 
 Currently, CSS `ellipsis` is not supported. However, the string length can be limiting by using JavaScript. For an example code, see the Scrolling Text sample in [Animation](./app-management/web-widget-ww.md#animation).
```
Comment (mijin85cho) [inline docs/application/web/guides/app-management/web-widget.md]: `./app-management/web-widget-ww.md#animation` should be changed into `#animation`

File: docs/application/web/guides/connectivity/download.md [hunk 14]
```diff
@@ -1,24 +1,18 @@
...
 
 - Checking the download state and information
 
-  You can [access the current download state and retrieve the download information](./connectivity/download-w.md#check) using the `DownloadManager` interface. The states are defined in the `DownloadState` enumerator (in [mobile](../../../../org.tizen.web.apireference/html/device_api/mobile/tizen/download.html#DownloadState), [wearable](../../../../org.tizen.web.apireference/html/device_api/wearable/tizen/download.html#DownloadState), and [TV](../../../../org.tizen.web.apireference/html/device_api/tv/tizen/download.html#DownloadState) applications).
+  You can [access the current download state and retrieve the download information](#checking-the-download-state-and-information) using the `DownloadManager` interface. The states are defined in the `DownloadState` enumerator (in [mobile](../../../../org.tizen.web.apireference/html/device_api/mobile/tizen/download.html#DownloadState), [wearable](../../../../org.tizen.web.apireference/html/device_api/wearable/tizen/download.html#DownloadState), and [TV](../../../../org.tizen.web.apireference/html/device_api/tv/tizen/download.html#DownloadState) applications).
 
 ## Prerequisites
```
Comment (mijin85cho) [inline docs/application/web/guides/connectivity/download.md]: In the line 35, `[virtual root location](../data/file_system_w.htm#roots)` should be changed into `[virtual root location](file-system.md#roots)`

File: docs/application/web/guides/connectivity/nfc.md [hunk 15]
```diff
@@ -492,17 +487,17 @@ When an application no longer needs to receive card emulation events, change the
 
 ## NFC Application Control Operations
 
-You can launch NFC applications based on the NDEF message content using the [application control](./app-management/app-controls-w.md) functionalities:
+You can launch NFC applications based on the NDEF message content using the [application control](./app-management/app-controls.md) functionalities:
```
Comment (mijin85cho) [inline docs/application/web/guides/connectivity/nfc.md]: It looks like not ./app-management/app-controls.md, but **..**/app-management/app-controls.md.

File: docs/application/web/guides/cordova/globalization.md [hunk 16]
```diff
@@ -16,18 +10,18 @@ The main features of the Globalization API include:
...
 
-  - [Get the user's current language](./cordova/globalization-w.md#current_language) as the BCP 47 identifier (such as en-US).
-  - [Get the user's current locale](./cordova/globalization-w.md#current_locale) as the BCP 47 identifier (such as en-US).
-  - [Get the pattern string to format and parse currency](./cordova/globalization-w.md#currency) and the ISO 4217 currency code.
-  - [Get the names of months and names of the days of the week](./cordova/globalization-w.md#names_of_months_and_days).
-  - [Get the pattern string to format and parse dates](./cordova/globalization-w.md#date_format).
-  - [Get the first day of the week](./cordova/globalization-w.md#first_day_of_week).
-  - [Get the pattern string to format and parse numbers](./cordova/globalization-w.md#number_format).
-  - [Determine whether the Daylight Saving Time is in effect](./cordova/globalization-w.md#dst) for a given date.
+  - [Get the user's current language](#retrieving-the-current-kanguage) as the BCP 47 identifier (such as en-US).
```
Comment (mijin85cho) [inline docs/application/web/guides/cordova/globalization.md]: It looks like not `#retrieving-the-current-kanguage` but `#retrieving-the-current-language`

**Comment by journey2w** [comment]
I will apply your comment to the next PR.

File: docs/application/web/guides/cordova/globalization.md (hunk 16 above)
Comment (journey2w) [reply docs/application/web/guides/cordova/globalization.md]: done

File: docs/application/web/guides/connectivity/nfc.md (hunk 15 above)
Comment (journey2w) [reply docs/application/web/guides/connectivity/nfc.md]: done

File: docs/application/web/guides/app-management/app-management-cover.md (hunk 11 above)
Comment (journey2w) [reply docs/application/web/guides/app-management/app-management-cover.md]: I will remove app-management-cover.md but applications.md will substitue this page's role.

File: docs/application/web/guides/app-management/app-controls.md (hunk 10 above)
Comment (journey2w) [reply docs/application/web/guides/app-management/app-controls.md]: Done

File: docs/application/web/guides/TOC.md (hunk 8 above)
Comment (journey2w) [reply docs/application/web/guides/TOC.md]: already done.

File: docs/application/web/guides/TOC.md (hunk 7 above)
Comment (journey2w) [reply docs/application/web/guides/TOC.md]: It was discussed with a maintainer. watch-app.md will be removed.

File: docs/application/web/getting-started/wearable-watch/first-app-watch.md (hunk 6 above)
Comment (journey2w) [reply docs/application/web/getting-started/wearable-watch/first-app-watch.md]: Done

File: docs/application/web/getting-started/wearable/first-app.md (hunk 5 above)
Comment (journey2w) [reply docs/application/web/getting-started/wearable/first-app.md]: already done

File: docs/application/web/guides/app-management/web-widget.md (hunk 13 above)
Comment (journey2w) [reply docs/application/web/guides/app-management/web-widget.md]: done

**Pull request merged**